### ISSUE TITLE
UPM Support

### ISSUE THREAD

**Issue by Lawendt** [snozbot/fungus#826]
Hey, I wanted to know if you guys are looking towards UPM support in a near future.

I don't know what you are thinking of ways to add this workflow, I've seen some people, myself included, using [github ci workflows](https://github.com/Lawendt/LawendtUnityUtilities/blob/master/.github/workflows/upm.yml). This might be useful.

**Comment by stevehalliwell** [comment]
Yep, only had a cursory glance over it at the moment. It'll most likely be looked at more closely shortly after 3.13 is released. Following the [openupm guide](https://openupm.com/docs/adding-upm-package.html#upm-package-criteria), I imagine. If you have any other thoughts or advice, let us know.

**Comment by stevehalliwell** [comment]
Noting that it appears from https://docs.unity3d.com/Manual/CustomPackages.html that switching to the recommended layout may not be compatible with anything below 2019.1
